src/core/packages/security/server/src/authz.ts

```diff
@@ -7,6 +7,29 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
+export enum AuthzOptOutReason {
+  DelegateToESClient = 'Route delegates authorization to the scoped ES client',
+  DelegateToSOClient = 'Route delegates authorization to the scoped SO client',
+  ServeStaticFiles = 'Serving static files that do not require authorization',
```

Comment:
Minor nit - match the format and tense of the other messages above
```suggestion
  ServeStaticFiles = 'Route serves static files that do not require authorization',
```